config/schema/oe_translation.schema.yml

```diff
@@ -30,3 +30,14 @@ oe_translation.oe_translation_request_type.*:
     label:
       type: label
       label: 'Label'
+
+field.value.oe_translation_language_with_status:
```

Comment:
We need to update the schema for existing sites cause the Local translations dashboard is not accessible now.

Reply:
It's not that. It was caused from before, fixed by fixing the entity definition creation in the update hook.